src/openloco/windows/LandscapeGeneration.cpp

```diff
@@ -1179,9 +1179,9 @@ namespace openloco::ui::windows::LandscapeGeneration
             widgets[widx::num_industries].text = numIndustriesLabels[s5::getOptions().numberOfIndustries];
 
             window->activated_widgets &= ~((1 << widx::check_allow_industries_close_down) | (1 << widx::check_allow_industries_start_up));
-            if (industryFlags & scenario::industry_flags::allow_industries_close_down)
+            if ((industryFlags & scenario::industry_flags::disallow_industries_close_down) == 0)
```

Comment:
is this the preferred style for the code base? I much prefer `!(industryFlags & scenario::industry_flags::disallow_industries_close_down)` I find the `== 0` and `!= 0` is much easier to miss and requires more thought.

Reply:
I'm finding 730 occurrences of ` [&|] `, only 176 of which (24.1%) occur along an (in)equals operator. Admittedly, this surprises me a little.

For the case of zero vs non-zero I have no objection to using the negation operator. Like you, I wasn't sure what we settled on, though — I thought we were doing the same thing as OpenRCT2. We should really write up a code-style soon.